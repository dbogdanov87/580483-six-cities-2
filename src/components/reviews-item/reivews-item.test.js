import React from "react";
import renderer from "react-test-renderer";
import ReviewsItem from "./reivews-item.jsx";

it(`Reviews Item is rendered correctly`, () => {

  const div = document.createElement(`div`);
  div.id = `map`;
  document.body.appendChild(div);

  const review = {
    id: 1,
    avatarImage: `str`,
    userName: `str`,
    rating: `str`,
    reviewsText: `str`,
    dateTime: `str`,
    dateString: `str`
  };

  const tree = renderer.create(<ReviewsItem
    review={review}
  />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
