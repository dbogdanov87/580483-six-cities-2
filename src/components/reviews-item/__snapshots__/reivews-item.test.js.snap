// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reviews Item is rendered correctly 1`] = `
<li
  className="reviews__item"
>
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Reviews avatar"
        className="reviews__avatar user__avatar"
        height="54"
        src="str"
        width="54"
      />
    </div>
    <span
      className="reviews__user-name"
    >
      str
    </span>
  </div>
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": "str",
            }
          }
        />
        <span
          className="visually-hidden"
        >
          Rating
        </span>
      </div>
    </div>
    <p
      className="reviews__text"
    >
      str
    </p>
    <time
      className="reviews__time"
      dateTime="str"
    >
      str
    </time>
  </div>
</li>
`;
